Makefile

```diff
@@ -205,7 +205,29 @@ clean:
 distclean: clean
 	-rm -rf vendor
 
+modules:
+	-go mod tidy
+
+verify-modules:
+	-rm -f go.mod
+	-go mod init
+	-go mod tidy
+	# This line checks that we haven't changed the go.mod or go.sum file apart from the fist line
+	# (because Travis thinks that the local build is under the _user's own_ module)
```

Comment:
Also, these two lines will show up in the make output since you have tabs in front of them... they are each essentially a bash script consisting of a commented line. You could silence them with a leading `@` character, or indent them using zero-or-more space characters instead of tabs.